src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -424,7 +443,7 @@ abstract protected function getMappingObjectDefaultName();
      *
      * @return string
      */
-    abstract protected function getMappingResourceConfigDirectory();
+    abstract protected function getMappingResourceConfigDirectory(/* string $bundleDir */);
```

Comment:
the new argument must have a default value to be a proper BC layer, and we should document it with `@param`